server/src/main/java/io/littlehorse/server/monitoring/metrics/ServerFilterRules.java

```diff
@@ -9,6 +9,7 @@ private ServerFilterRules() {}
 
     // relevance ordered
     public static final List<ServerFilterRule> RULES = List.of(
+            // TODO: Wait for KIP-869 and gather state restoration metrics.
```

Comment:
here we should remove all the custom metric we created before and are removed now

Reply:
Roger Roger